src/server/api/routers/members.ts

```
@@ -3,6 +3,7 @@
 import { TRPCError } from '@trpc/server'
 import { getTranslations } from 'next-intl/server'
 import { z } from 'zod'
+import crypto from 'crypto'
```

ESLint

[error] `crypto` import should occur before import of `@trpc/server`

(import/order)